---
title: Lab 4 - Data Analysis
---

::: {.todo title="Lab 4 - Data Analysis Tasks"}

After attending the practical lab, you should:

* Try to identify your unknown strain (Case Study 5) as precisely as possible, using

  * the results from your MSA plates and biochemical tests (@sec-4ABdata)

  * the results from your MLVA experiment (@sec-MLVAdata)

* Try to identify the unknown from Case Study 3 (the contaminated spinach), combining the results from lab 3 with the results from your MacConkey Agar plates (@sec-Macdata)

* Try and identify the unknown from Case Study 4 (the careless clinician), using the results from your thioglycollate cultures and your anaerobically incubated plates (@sec-4Ddata) 

:::

## Task 4A and Task 4B Data Analysis {#sec-4ABdata}

For the controls (<i>Staphylococcus aureus</i> and <i>Staphylococcus epidermidis</i>) - does the phenotype you observed on Mannitol Salt Agar match what you would expect based on the literature?

How precisely can you identify your unknown based on your observations (phenotype on MSA, lysostaphin sensitivity (performed for you), coagulase test results? 

## MLVA Data Analysis (Task 4C) {#sec-MLVAdata}

The predicted size of each PCR product is described in @tbl-mlva. Note that these were calculated using the N315 reference genome sequence - the size of each PCR product will differ depending on the number of VNTRs at that locus. 

| Gene/locus | Product size* (bp) |
|------|------|
| clfA | 1183 |       
| clfB | 828  |       
| sdr  | 670  |       
| spa  | 284  |       
| ssp  | 173  | 

: Predicted PCR product size for S. aureus MLVA typing. *Product size is estimated with reference to genome N315	 {#tbl-mlva}

Class data for the MLVA results are shown in @fig-MLVAresults. 

![S. aureus MLVA results. Lane 1: SA98; Lane 2: SA130; Lane 3: SA82; Lane 4: SA117; Lane 5: unknown 1; Lane 6: unknown 2; Lane 7: unknown 3; Lane 8: 100bp ladder](images/MLVAresults.png){#fig-MLVAresults}

THe <a href="https://www.promega.co.uk/products/cloning-and-dna-markers/dna-ladder-rna-ladder/100bp-dna-ladder/?catNum=G2101">Promega 100 bp ladder</a> was run alongside the PCR products to allow you to estimate the size of each band.^[You could, of course, create a DNA standard curve and use this to more precisely determine the size of each band.] 

* Are you able to use the four known <i>S. aureus</i> strains (SA98, SA130, SA82, and SA117) to more precisely identify the three unknowns?

* What other experiments could you do to definitively identify these unknowns?

* What do your results imply for Case Study 5? Is the doctor in question guilty of poor infection control measures (the source of the epidemic), or do you believe the strains are community-acquired (from different sources)? What would be your recommendation(s) as a consulting microbiologist?


## Case Study 3 - unknown identification {#sec-Macdata}

For the controls (<i>Escherichia coli</i> and <i>Salmonella typhimurium</i>) - does the phenotype you observed on MacConkey Agar match what you would expect based on the literature?

For your unknown, does the phenotype you observed on MacConkey Agar align with the results of your API-20E strip test? (Note that *one* of the API-20E tests measures the activity of an enzyme important for the phenotype observed on MacConkey - which one?)

Identify your unknown as precisely as possible, based on the biochemical tests and API-20E strip test from the last lab, and the phenotype on MacConkey agar. For Case Study 3, what is the contaminant found on the spinach? What should the Food Standards Agency do?

## Case Study 4 - unknown identification {#sec-4Ddata}

For the fluid thioglycollate cultures, and the stiff blood agar plates incubated anoxically, do the phenotypes of the controls (<i>Escherichia coli</i> and <i>Clostridium</i> sp.) match what you would expect based on the literature?

Can you identify your unknown based on its phenotypes? 

What further experiments could you do (to identify the unknown more precisely, or confirm its identification)?

::: {#discussion-streak .discussion title="Lab 4 Discussion Questions"}

**Questions to consider/discuss with your lab partners:**

1. As a clinical microbiologist, you wish to type a number of newly isolated <i>S. aureus</i> strains using MLVA. However, a careless technician threw the cultures and samples away after you had deposited the isolates' genome sequence data in the NCBI database. Without the cultures available, is there any way you could determine the MLVA types of these isolates?

2. When setting up a set of MLVA PCRs, you inadvertently omitted two primers: the forward primer for amplifying <i>cflA</i>, and the reverse primer for amplifying <i>cflB</i>. If there are no other technical issues/problems with the PCR, how many bands will you observe when you run these reactions out on an agarose gel?

3. If the seal on the GasPak chamber failed, and the plates within were exposed to oxygen during their incubation period, what phenotypes would you expect to observe for your controls and unknown?

4. If your lab has run out of MacConkey/lactose medium, and you streak your controls and unknown organism on MacConkey/glucose medium instead, what phenotypes will you observe for each organism?

5. Are there any other sugars that could be added to MacConkey agar that would be diagnostically useful for a clinical microbiologist?

:::

::: {.discussion-hints title="Hints - Lab 4 Discussion Questions" collapse="true"}

1. Can you predict the size of a PCR product if given the sequence of the genomic DNA to be used as a template, and the sequences of the PCR primers?

2. Neither of the <i>cfl</i> PCRs will work, because of the missing primers.

3. The indicator strip in the GasPak chamber will remain blue in the presence of oxygen, and the strains should have the same phenotypes as you observed for the plates that were incubated outside of the GasPak chamber.

4. Are the organisms able to ferment glucose?

5. Consider the sugars used in the API-20E strip test...

:::